acceptable/__main__.py

```diff
@@ -146,5 +171,40 @@ def render_markdown(metadata, name):
     )
 
 
+def lint_cmd(cli_args, stream=sys.stdout):
+    sys.path.insert(0, os.getcwd())
+    current, locations = import_metadata(cli_args.modules)
+    try:
+        metadata = json.load(cli_args.metadata)
+    except json.JSONDecodeError as e:
+        return 'Error parsing {}: {}'.format(cli_args.metadata.name, e)
+    finally:
+        cli_args.metadata.close()
+
+    has_errors = False
+    display_level = lint.WARNING
+    error_level = lint.DOCUMENTATION
+
+    if cli_args.strict:
+        display_level = lint.WARNING
+        error_level = lint.WARNING
+    elif cli_args.quiet:
+        display_level = lint.DOCUMENTATION
+
+    for message in lint.metadata_lint(metadata, current, locations):
+        if message.level >= display_level:
+            stream.write('{}\n'.format(message))
+
+        if message.level >= error_level:
+            has_errors = True
+
+    if cli_args.update:
+        if not has_errors or cli_args.force:
+            with open(cli_args.metadata.name, 'w') as f:
+                f.write(json.dumps(current, indent=2, sort_keys=True))
```

Comment:
instead of `f.write(json.dumps(current, ...))` just use `json.dump(current, f, ...)`